linux_os/guide/services/avahi/disable_avahi_group/package_avahi-autoipd_removed/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Uninstall avahi-autoipd Server Package'
+
+description: |-
+    If the system does not need to have an Avahi server which implements 
+    the DNS Service Discovery and Multicast DNS protocols,
+    the avahi-autoipd and avahi packages can be uninstalled.
+
+rationale: |-
+    Automatic discovery of network services is not normally required for 
+    system functionality. It is recommended to remove this package to reduce 
+    the potential attack surface.
+
+severity: medium
+
+identifiers:
+    cce@sle12: CCE-92310-2
+    cce@sle15: CCE-92465-4
+    
+references:
```

Comment:
Are you sure that all these references are valid for these new rules? Have you confirmed them? If not, I would suggest to keep only the confirmed references.